locales/pdk.pot

```diff
@@ -6,11 +6,11 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: puppet development kit v1.9.1-46-g3e22a80\n"
+"Project-Id-Version: puppet development kit v1.10.0-82-gd468eda\n"
 "\n"
 "Report-Msgid-Bugs-To: docs@puppet.com\n"
-"POT-Creation-Date: 2019-04-02 10:51+1100\n"
```

Comment:
Maybe for another day, but localisation may be better as a different commit to avoid the amount of diff spam.